content/reference/admin/custom_sso/index.md

```diff
@@ -61,7 +61,12 @@ You will need to edit in the application integration you created to configure th
 If your SSO system supports both OIDC and SAML, we recommend using OIDC.
 {{< /alert >}}
 
-If your SSO system offers SAML v2.0 integration, it will typically publish its Identity Provider (IdP) entity descriptor on a specific URL. To configure the SSO integration, we will need this IdP entity descriptor URL.
+If your SSO system offers SAML v2.0 integration, it will typically publish its Identity Provider (IdP) entity descriptor on a specific URL.
```

Comment:
Maybe explain the 2 cases: public SSO system that our platform can connect to, and private SSO system that is only accessible on the intranet/VPN, in which case we need an export of the mapping?